### PR TITLE
Feature/dvop 3113 company accounts api ecs terraform update

### PR DESCRIPTION
**Pull request by cryder-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: IaC Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

File: terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars [hunk 1]
```diff
@@ -2,4 +2,10 @@ environment = "cidev"
 aws_profile = "development-eu-west-2"
 
 # service configs
-use_set_environment_files = true
+log_level = "info"
```
Comment (makhtar-ch) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:5]: If the log_level is defined in the [S3](https://github.com/companieshouse/ecs-service-configs-dev/blob/main/development-eu-west-2/cidev/company-accounts.api.ch.gov.uk.env) .env file, the log_level from these profile/vars can be removed.

Comment (makhtar-ch) [inline terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars:5]:
```suggestion

```
If the log_level is defined in the S3 .env file, the log_level from these profile/vars can be removed.

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars [hunk 2]
```diff
@@ -2,4 +2,6 @@ environment = "live"
 aws_profile = "live-eu-west-2"
 
 # service configs
-use_set_environment_files = true
+log_level = "info"
```
Comment (makhtar-ch) [inline terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:5]:
```suggestion

```
same here

File: terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars [hunk 3]
```diff
@@ -2,9 +2,9 @@ environment = "staging"
 aws_profile = "staging-eu-west-2"
 
 # service configs
-use_set_environment_files = true
+log_level = "info"
```
Comment (makhtar-ch) [inline terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars:5]:
```suggestion

```
same here

**Review by makhtar-ch — CHANGES_REQUESTED** [review]

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Comment by ch-code-analysis** [comment]
CI: No IaC security warnings found

**Review by makhtar-ch — APPROVED** [review]

**Pull request merged**